src/test/scala/uk/gov/hmrc/mongo/ReactiveRepositorySpec.scala

```diff
@@ -136,7 +136,10 @@ class ReactiveRepositorySpec extends WordSpec with Matchers with MongoSpecSuppor
 
       await(repository.save(e1))
 
-      await(repository.removeById(e1.id)).inError shouldBe false
+      val removeResult = await(repository.removeById(e1.id))
+      val inError = !removeResult.ok || removeResult.code.isDefined
+
+      inError shouldBe false
```

Comment:
`inError` is not a private method so I just used the logic from the definition to create a local `val` to use it for tests.